.github/actions/integration-tests/run-ci-stage1

```diff
@@ -61,6 +61,11 @@ fi
 eval set -- "${opts}"
 while true; do
     case "${1}" in
+        --ci-param-mode)
+            shift
+            CI_PARAM_MODE="${1}"
+            shift
+            ;;
```

Comment:
I would prefer a default set above like all the other parameters.

Reply:
ack